requirements.txt

```diff
@@ -6,7 +6,8 @@ Flask-Script==2.0.5
 Flask-SQLAlchemy==2.0
 psycopg2==2.5.4
 jsonschema==2.3.0
-git+https://github.com/alphagov/digitalmarketplace-utils.git@0.14.0#egg=digitalmarketplace-utils==0.14.0
+git+https://github.com/pcraig3/Flask-FeatureFlags.git@master#egg=Flask-FeatureFlags==0.6-dev
+git+https://github.com/alphagov/digitalmarketplace-utils.git@0.17.0#egg=digitalmarketplace-utils==0.17.0
```

Comment:
I think it's probably worth pushing a new version tag for 0.18.0 and updating these pull-requests to use it.